mineral-ui/mineral-ui
# [Website] fill in content for Menu and Dropdown

### Description
- add content for Menu, MenuItem, MenuDivider, MenuGroup, Dropdown
- hide tab interface if there is only one tab on docs pages
- update snapshots

closes #377 
closes #378 

### Motivation and context
we separated the feature [PR for Menu and Dropdown](https://github.com/mineral-ui/mineral-ui/pull/363) from this content PR.

### How to test
1. `cd mineral-ui && npm start`
2. Check the Dropdown, Menu, MenuItem, MenuDivider, and MenuGroup pages for lede, examples, when/how to use, and dos/don'ts
3. This was in the middle of adding a trailing slash to the navigation, so make sure all my links to the other component pages work.

### Types of changes
- docs

### Checklist




### How does this PR make you feel?

![what's on the menu](https://media.giphy.com/media/3o7TKMlJrVQ5ket3hu/giphy.gif)
